packages/dependency-extraction-webpack-plugin/index.js

```diff
@@ -98,27 +110,31 @@ class DependencyExtractionWebpackPlugin {
 					}
 				}
 
-				// Build a stable JSON string that declares the WordPress script dependencies.
-				const sortedDepsArray = Array.from( entrypointExternalizedWpDeps ).sort();
-				const depsString = JSON.stringify( sortedDepsArray );
+				const runtimeChunk = entrypoint.getRuntimeChunk();
+
+				// Build a stable JSON string that declares the WordPress script asset.
```

Comment:
May not be JSON
```suggestion
				// Get a stable, serialized representation of the WordPress script asset.
```

Reply:
I opted for:

```js
// Get a stable, stringified representation of the WordPress script asset.
```
I renamed the method to `this.stringify` to align with that.